<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STGEditor</class>
 <widget class="QWidget" name="STGEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="ruleComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="canonicalNameListWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="generalSettingsPushButton">
       <property name="text">
        <string>Generation Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,3,3,0">
     <item>
      <widget class="QGroupBox" name="aliasGroupBox">
       <property name="title">
        <string>Alias</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="STGAliasListWidget" name="aliasListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="STGFragmentInputWidget" name="headerFragmentInputWidget" native="true"/>
     </item>
     <item>
      <widget class="STGFragmentInputWidget" name="delimiterFragmentInputWidget" native="true"/>
     </item>
     <item>
      <widget class="STGFragmentInputWidget" name="tailFragmentInputWidget" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="dtEnableCheckBox">
         <property name="text">
          <string>Show all fields</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>STGFragmentInputWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/SimpleTextGenerator/STGFragmentInputWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>STGAliasListWidget</class>
   <extends>QListWidget</extends>
   <header>src/gui/SimpleTextGenerator/STGAliasListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
